corehq/apps/userreports/reports/factory.py

```diff
@@ -71,6 +73,7 @@ class ReportColumnFactory(object):
         'percent': PercentageColumn,
         'sum_when': SumWhenColumn,
         'sum_when_template': SumWhenTemplateColumn,
+        AGGGREGATION_TYPE_ARRAY_AGG_LAST_VALUE: ArrayAggLastValueReportColumn,
```

Comment:
I actually think this is the right way to do this, just like `SQLAGG_COLUMN_MAP` is defined.
I guess the way above or like
`ArrayAggLastValueReportColumn.type: ArrayAggLastValueReportColumn`

We have the same string declared as a constant and then used at other places which does not sound right to me.

Reply:
> We have the same string declared as a constant and then used at other places which does not sound right to me.

Agreed.  I commented since I saw that `AGGGREGATION_TYPE_ARRAY_AGG_LAST_VALUE` is only used here.  I didn't realize that it's the same as `ArrayAggLastValueReportColumn.type`.  I agree with you that having all of these reference `MyColumn.type` would be more ideal.